src/pages/homepage.tsx

```diff
@@ -93,117 +101,213 @@ const HomePage = () => {
-            link="https://tosdr.org"
-            center={true}
-          ></Card>
-          <Card
-            image="/images/reuses/readability.jpg"
-            title={t('homepage:reuses.readability.title')}
-            subtitle={t('homepage:reuses.readability.subtitle')}
-            author={t('homepage:reuses.readability.author')}
-            link="https://disinfo.quaidorsay.fr/en/open-terms-archive/experiments"
-            center={true}
-          ></Card>
-          <Card
-            image="/images/reuses/scripta-manent.jpg"
-            title={t('homepage:reuses.scripta_manent.title')}
-            subtitle={t('homepage:reuses.scripta_manent.subtitle')}
-            author={t('homepage:reuses.scripta_manent.author')}
-            link="https://disinfo.quaidorsay.fr/fr/open-terms-archive/scripta-manent"
-            center={true}
-          ></Card>
-          <Card
-            image="/images/reuses/tosback.jpg"
-            title={t('homepage:reuses.tosback.title')}
-            subtitle={t('homepage:reuses.tosback.subtitle')}
-            author={t('homepage:reuses.tosback.author')}
-            link="https://tosback.org/"
-            center={true}
-          ></Card>
-          <Card
-            image="/images/reuses/new-reuse.jpg"
-            className="text__center"
-            title={t('homepage:reuses.new_reuse.title')}
-            subtitle={t('homepage:reuses.new_reuse.subtitle')}
-            link="mailto:contact@opentermsarchive.org"
-            center={true}
-          ></Card>
-        </CardList>
+
+      {/* Product newsletter */}
+      <Container layout="wide" dark={true} paddingY={false}>
+        <Container gridCols="9" gridGutters="8" flex={true}>
+          <Column width={40} alignX="center" alignY="center">
+            <IconMail size="128" color="var(--colorBlack400)" strokeWidth="1" />
+          </Column>
+          <Column width={60} subtitle={t('homepage:product-newsletter.title')}>
+            <TextContent className="mt__0">
+              <p>{t('homepage:product-newsletter.desc')}</p>
+              <form
+                id="sib-form"
+                method="POST"
+                action={
+                  router.locale === 'fr'
+                    ? 'https://98bb6346.sibforms.com/serve/MUIEAMjUWLVIdeiQhwdN4AJo1SB8QMtDFKbp-JIARujMmZ9_feY8x0U6V84om-RPrHKu22mPwcozi9kjLI8gP1bMWgqFwsmw_7Y5rDuFHMGEGz5JqMhgYzcFoO78g8zvArqGLojk4L_6mm4cz3y9c3LkVMrrgs32BFinj1yIBdN5knTmlmbhsOYc2OhwVLFRkRTYg2luE4LwWHyV'
+                    : 'https://98bb6346.sibforms.com/serve/MUIEABuvRacGvFG5qloccLUI_5pWT2w-cVI7L5RogU08EvZ_-d1aTmfUQd-XrwdNYa4MSZbn09Yq-px0kkqKuzbQRjJs9KIzYPwHEbs-3YCnczbaQq4hQthHw0hKh2BcajLr98vsJni7IzUuQRjz1rb6vemMlGn1-raTiNiDxy5Ujm69CxV_wjMFTr4BiSiSAUBiLpOA0Vv3zYcG'
+                }
+                data-type="subscription"
+              >
+                <div className="formfield mb__L">
+                  <input
+                    type="text"
```

Comment:
Let the browser help users with autofill and validation 🙂 

```suggestion
                    type="email"
```

Reply:
Done with https://github.com/ambanum/OpenTermsArchive.org/pull/161/commits/4b0cea8844aa47ec5c7512fe484244d4080d5d33